steembase/http_client.py

```diff
@@ -139,15 +142,30 @@ def json_rpc_body(name, *args, api=None, as_json=True, _id=0, kwargs=None):
             Otherwise, a Python dictionary is returned.
 
         """
+        api = kwargs.get('kwargs').pop('api', None)
+        as_json = kwargs.get('kwargs').pop('as_json', True)
+        _id = kwargs.get('kwargs').pop('_id', 0)
+
+        if len(kwargs) == 1 and 'kwargs' in kwargs:
+            kwargs = None
```

Comment:
What's the other cases here? Is it possible for `len(kwargs) == 1` but not `'kwargs' in kwargs`? Or `len(kwargs) > 1` *and* `'kwargs' in kwargs`?

If there's some condition we always expect (or assume) here let's add asserts. Otherwise I may be missing something but it looks a bit cryptic to me.